### ISSUE TITLE
Add 'workspace' or 'team' grouping

### ISSUE THREAD

**Issue by ryanm101** [nosarthur/gita#71]
I work across multiple teams on the overall product, but each team has its own set of repos. Being able to create a grouping so that `gita ll` or `gita ls` would only show the repos in that group/team/workspace would bbe useful.

E.g.
```
T1_repo1
T1_repo2
T1_repo3
```
```
T2_repo1
T2_repo2
T2_repo3
T2_repo4
```

`gita addteam t1`
`gita addteam t2`
`gita setteam t1`

`gita ls`
```
T1_repo1
T1_repo2
T1_repo3
```

`gita setteam t2`

`gita ls`
```
T2_repo1
T2_repo2
T2_repo3
T2_repo4
```

**Comment by ryanm101** [comment]
I work across multiple teams on the overall product, but each team has its own set of repos. Being able to create a grouping so that `gita ll` or `gita ls` would only show the repos in that group/team/workspace would bbe useful.

E.g.
```
T1_repo1
T1_repo2
T1_repo3
```
```
T2_repo1
T2_repo2
T2_repo3
T2_repo4
```

**Comment by nosarthur** [comment]
It will be addressed in #69
